### PR TITLE
Add links to known issues from json schema

### PR DESCRIPTION
**Pull request by mccalluc**

### CODE REVIEW

File: context/app/validation_utils.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import jsonschema
+
+
+def for_each_validation_error(data, schema, callable):
```
Comment (ilan-gold) [inline context/app/validation_utils.py]: Maybe add some comments as to what is going on here with respect to why we are appending `_TODO`

File: context/app/routes_main.py [hunk 2]
```diff
@@ -44,11 +44,7 @@ def details(type, uuid):
 
     entity = client.get_entity(uuid)
     entity_schema = load_yaml(open(current_app.root_path + '/schemas/entity.yml'))
```
Comment (ilan-gold) [inline context/app/routes_main.py]: maybe the `entity.yml` should be a variable set at the top of this file.  also, maybe a `with open(...)`

File: context/app/validation_utils.py (hunk 1 above)
Comment (mccalluc) [reply context/app/validation_utils.py]: yes: done.

File: context/app/routes_main.py (hunk 2 above)
Comment (mccalluc) [reply context/app/routes_main.py]: Added `with open`. Tried moving it up and out, but then `current_app` isn't available. Not sure about the best approach, but just added a TODO for now.

Comment (ilan-gold) [reply context/app/routes_main.py]: ok - i meant just the `'/schemas/entity.yml'` as a variable since the `current_app` shouldn't change.

**Pull request merged**